engine/helpers_test.go

```diff
@@ -1021,27 +1022,20 @@ func TestGetCryptocurrencyDepositAddressesByExchange(t *testing.T) {
 func TestGetExchangeCryptocurrencyDepositAddress(t *testing.T) {
 	t.Parallel()
 	e := createDepositEngine(&fakeDepositExchangeOpts{SupportsAuth: true, SupportsMultiChain: true})
+	_, err := e.GetExchangeCryptocurrencyDepositAddress(t.Context(), "non-existent", "", "", currency.BTC, false)
+	assert.ErrorIs(t, err, ErrExchangeNotFound)
+
 	const exchName = "fake"
-	if _, err := e.GetExchangeCryptocurrencyDepositAddress(context.Background(), "non-existent", "", "", currency.BTC, false); !errors.Is(err, ErrExchangeNotFound) {
-		t.Errorf("received %s, expected: %s", err, ErrExchangeNotFound)
-	}
-	r, err := e.GetExchangeCryptocurrencyDepositAddress(context.Background(), exchName, "", "", currency.BTC, false)
-	if err != nil {
-		t.Error(err)
-	}
-	if r.Address != "fakeaddr" {
-		t.Error("unexpected address")
-	}
+	r, err := e.GetExchangeCryptocurrencyDepositAddress(t.Context(), exchName, "", "", currency.BTC, false)
+	require.NoError(t, err, "GetExchangeCryptocurrencyDepositAddress must not error")
+	assert.Equal(t, "fakeaddr", r.Address, "Address must be equal to fakeaddr")
```

Comment:
```suggestion
	assert.Equal(t, "fakeaddr", r.Address, "Address should be equal to fakeaddr")
```

Reply:
🦍 So long as we're here: Prefer to not stutter the expected condition (even true/false)
```suggestion
	assert.Equal(t, "fakeaddr", r.Address, "Should return the correct r.Address")
```